felix/Makefile

```diff
@@ -280,14 +285,14 @@ $(FELIX_CONTAINER_CREATED): register \
 UT_PACKAGES_TO_SKIP?=fv,k8sfv,bpf/ut
 
 .PHONY: ut
-ut combined.coverprofile: $(SRC_FILES) $(LIBBPF_A) build-bpf
+ut combined.coverprofile: $(SRC_FILES)  $(LIBBPF_A) build-bpf
 	@echo Running Go UTs.
 	$(DOCKER_RUN) -e $(ACK_GINKGO) $(CALICO_BUILD) ./utils/run-coverage -skipPackage $(UT_PACKAGES_TO_SKIP) $(GINKGO_ARGS)
 
 ###############################################################################
 # FV Tests
 ###############################################################################
-fv/fv.test: $(LIBBPF_A) $(SRC_FILES) $(FV_SRC_FILES)
+fv/fv.test: clone-libbpf $(LIBBPF_A) $(SRC_FILES) $(FV_SRC_FILES)
```

Comment:
I'd like to avoid any phony make dependencies on fv.test (I did some work to squash those a while ago and it made `make fv` a lot faster), can you depend on the "created file" instead so that fv.test will not always appear stale?